<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件传输</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-image: url(:/img/background.jpeg);


}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>12</y>
      <width>741</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="fileMode">
         <property name="styleSheet">
          <string notr="true">font:24pt &quot;Microsoft YaHei UI&quot;;
color:rgba(55,152,251,1);
font-size:24px;
line-height:67px;</string>
         </property>
         <property name="text">
          <string>文件</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="directorMode">
         <property name="styleSheet">
          <string notr="true">font:24pt &quot;Microsoft YaHei UI&quot;;
color:rgba(55,152,251,1);
font-size:24px;
line-height:67px;</string>
         </property>
         <property name="text">
          <string>文件夹</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font:24pt &quot;Microsoft YaHei UI&quot;;
color:rgba(55,152,251,1);
font-size:24px;
line-height:67px;</string>
         </property>
         <property name="text">
          <string>文件（夹）路径:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="showFilePath"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font:24px &quot;Microsoft YaHei UI&quot;;
background-color:#3385ff;
color:rgba(255,255,255,1);
background:rgba(78,162,255,1);
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTextEdit" name="showUrlText"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">font:36px &quot;Microsoft YaHei UI&quot;;
background-color:#3385ff;
color:rgba(255,255,255,1);
background:rgba(78,162,255,1);
border-radius:15px;</string>
         </property>
         <property name="text">
          <string>生成二维码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font:36px &quot;Microsoft YaHei UI&quot;;
background-color:#3385ff;
color:rgba(255,255,255,1);
background:rgba(78,162,255,1);
border-radius:15px;</string>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>软件说明</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>软件版本</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectFilePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>663</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>generateQRCode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>-10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonGroup</sender>
   <signal>buttonClicked(QAbstractButton*)</signal>
   <receiver>MainWindow</receiver>
   <slot>selectFileMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectFilePath()</slot>
  <slot>generateQRCode()</slot>
  <slot>closeWindow()</slot>
  <slot>selectFileMode()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
